src/main/scala/scorex/core/network/NetworkController.scala

```diff
@@ -38,7 +41,10 @@ class NetworkController(settings: NetworkSettings,
 
   private implicit val system: ActorSystem = context.system
 
-  private val featureSerializers: PeerFeature.Serializers = features.map(f => f.featureId -> f.serializer).toMap
+  val a  = LocalAddressPeerFeature.featureId -> LocalAddressPeerFeatureSerializer
```

Comment:
`a` is a very bad name. Also it should by private as soon as other members are also private